_episodes/03-ncbi-sra.md

```diff
@@ -17,7 +17,15 @@ There are many repositories for public data. Some model organisms or fields have
 
 # Accessing the original archived data
 
-The [sequencing dataset (from Tenaillon paper) adapted for this lesson](http://www.datacarpentry.org/organization-genomics/data/) was obtained from the [NCBI Sequence Read Archive](http://www.ncbi.nlm.nih.gov/sra) which is a large (>3 quadrillion basepairs as of 2014) repository for next-generation sequence data. Like many NCBI databases, it is complex and mastering its use is greater than the scope of this lesson. Very often, as in the Blount paper, there will be a direct link (perhaps in the supplemental information) to where on the SRA the dataset can be found. E.g. the link from the Tenaillon paper is [http://www.ncbi.nlm.nih.gov/sra?term=SRA026813](http://www.ncbi.nlm.nih.gov/sra?term=SRA026813)  
+The [sequencing dataset (from Tenaillon paper) adapted for this lesson](http://www.datacarpentry.org/organization-genomics/data/) was obtained from the [NCBI Sequence Read Archive](http://www.ncbi.nlm.nih.gov/sra) which is a large (>3 quadrillion basepairs as of 2014) repository for next-generation sequence data. Like many NCBI databases, it is complex and mastering its use is greater than the scope of this lesson. Very often there will be a direct link (perhaps in the supplemental information) to where on the SRA the dataset can be found. We have provided one link from the Tenaillon paper here: [http://www.ncbi.nlm.nih.gov/sra?term=SRA026813](http://www.ncbi.nlm.nih.gov/sra?term=SRA026813). See the figures below for how the data was formatted in the original paper. In this case, the authors provided a "PRJNA" accession which is a BioProject identifier. Accession numbers will often also start with either ERX, ERR, DRX, DRR, SRX, or SRR. 
```

Comment:
Instead of "(from Tenaillon paper)" could this be reworded as "(from Tenaillon et al. 2016)" or "(from *the* Tenaillon paper)"

Reply:
"(from Tenaillon et al. 2016)" is correct. So let's use that.

Reply:
Using two figures *would* be faster than clicking links, so I'm willing to use both. Still think they should be resized.

Reply:
To chage the size, @taylorreiter would want to switch from markdown formatting to html formating, something like:  `<img src="../fig/03_paper_header.png" width="200"/>`
and `<img src="../fig/03_acc_info.png" width="200"/>`. I'm not sure if 200 pixels will get the optimal width, so you could try a few different settings.

Reply:
I like the size and format, thank you for taking this up @hoytpr! I think it flows much better now too, thank you for hunting down the proper accession-in-paper-to-SRP path!